Mathlib/Order/Heyting/Basic.lean

```diff
@@ -124,36 +124,36 @@ theorem hnot_apply [∀ i, HNot (π i)] (a : ∀ i, π i) (i : ι) : (￢a) i =
 end Pi
 
 /-- A generalized Heyting algebra is a lattice with an additional binary operation `⇨` called
-Heyting implication such that `a ⇨` is right adjoint to `a ⊓`.
+Heyting implication such that `a ⇨ ·` is right adjoint to `a ⊓ ·`.
```

Comment:
Lean 4 expects parentheses around lambda functions written with `·` (here and elsewhere).
```suggestion
Heyting implication such that `(a ⇨ ·)` is right adjoint to `(a ⊓ ·)`.
```

Reply:
done!